src/client.ts

```diff
@@ -1,18 +1,24 @@
-import "fast-text-encoding";
-import { createClient } from "@dynamic-labs/client";
-import { ReactNativeExtension } from "@dynamic-labs/react-native-extension";
-import { ViemExtension } from "@dynamic-labs/viem-extension";
+import 'fast-text-encoding'
+import { createClient } from '@dynamic-labs/client'
+import { ReactNativeExtension } from '@dynamic-labs/react-native-extension'
+import { ViemExtension } from '@dynamic-labs/viem-extension'
 
-const environmentId = process.env.EXPO_PUBLIC_ENVIRONMENT_ID as string || '2762a57b-faa4-41ce-9f16-abff9300e2c9';
+const environmentId =
+  (process.env.EXPO_PUBLIC_ENVIRONMENT_ID as string) ||
+  '2762a57b-faa4-41ce-9f16-abff9300e2c9'
 
 if (!environmentId) {
-  throw new Error("EXPO_PUBLIC_ENVIRONMENT_ID is required");
+  throw new Error('EXPO_PUBLIC_ENVIRONMENT_ID is required')
 }
 
+// Leave this undefined to use the default dynamic api base url
+const apiBaseUrl = process.env.EXPO_PUBLIC_API_BASE_URL
```

Comment:
why you need this here?

the env id is in production redcoast

Reply:
makes it easier to test with preprod env ids. If clients don't provide it, it still defaults to the correct prod api